about.html

```diff
@@ -1,12 +1,109 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About the August Match</title>
+  <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="css/about.css">
 </head>
+
 <body>
+  <div id="mobileDisplay">
+    <header>
+      <section class="prenav">
+        <ul class="prenavul">
+          <li>
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+          </li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </section>
+      <nav class="mynav">
+        <img class="ccsummitlogo" src="resources/Ccsummit2015_logo.png" alt="CC Summit Logo">
+        <ul class="navul">
+          <li class="liGray">
+            <a class="mainAnchor" href="index.html">Home</a>
+          </li>
+          <li class="liGray">Program</li>
+          <li class="liGray">Join</li>
+          <li class="liGray">Sponsor</li>
+          <li class="liGray">News</li>
+          <li class="liRed">CC Campaign</li>
+        </ul>
+      </nav>
+    </header>
   
+    <div class="introStyle">
+      <i id="toolbar" class="fa fa-bars toolbarStyle" aria-hidden="true"></i>
+  
+      <section class="introTextStyle">
+        <p class="pStyle1">this upcoming June 27, 1835</p>
+        <h1 class="LatoBold h1Style">Honinbo Jowa plays Akaboshi Intetsu</h1>
+  
+        <section class="midParagraph">The legendary players will meet against each other in what promises to be an epic
+          and unforgettable Go
+          match, representing the houses of Honinbo and Inoue, respectively.</section>
+        <section class="LatoBold emailSection">
+          <p>Write to us for any further inquiries about the event fdezcaminero@gmail.com</p>
+        </section>
+      </section>
+    </div>
+  
+    <section class="textCenter">
+      <p class="LatoBold">See other Go matches from the Edo period</p>
+
+      <div class="redbar"></div>
+  
+      <p class="LatoRegular">Take a look at other Go matches</p>
+    </section>
+    <section>
+      <div class="eventCard">
+        <h4 class="eventstyle">1832</h4>
+  
+        <p>The 1832 Honinbo Tournament held in Moscow</p>
+      </div>
+      <div class="eventCard">
+        <h4 class="eventstyle">1831</h4>
+        <p>The 1831 Honinbo Tournament held in Santo Domingo</p>
+      </div>
+    </section>
```

Comment:
Great design @fdezcaminero 🤝

- [ ] Kindly ensure your design replicates the design given.
- Your design
![your about](https://user-images.githubusercontent.com/63917725/235016199-b5c0805f-b1cf-4136-953d-198f7a9929ff.PNG)
- Given design
![given about](https://user-images.githubusercontent.com/63917725/235016233-e13c632f-a8bc-4cc9-bfad-4181ce8f23dc.PNG)